src/otx/v2/adapters/torch/mmengine/engine.py

```diff
@@ -40,135 +73,190 @@ class MMXEngine(Engine):
     This class is a subclass of the otx.v2.api.core.engine.Engine class and provides additional functionality
     for training and evaluating PyTorch models using the MMEngine framework.
     """
+    default_config = Config(DEFAULT_CONFIG)
 
     def __init__(
         self,
         work_dir: str | Path | None = None,
-        config: dict | (Config | str) | None = None,
     ) -> None:
         """Initialize a new instance of the MMEngine class.
 
         Args:
             work_dir (Optional[Union[str, Path]], optional): The working directory for the engine. Defaults to None.
-            config (Optional[Union[dict, Config, str]], optional): The configuration for the engine. Defaults to None.
         """
         super().__init__(work_dir=work_dir)
         self.runner: Runner
         self.latest_model = {"model": None, "checkpoint": None}
         self.registry = MMEngineRegistry()
-        self._initial_config(config)
         self.dumped_config = Config({})
 
-    def _initial_config(self, config: dict | (Config | str) | None) -> None:
-        if config is not None:
-            if isinstance(config, str):
-                self.config = Config.fromfile(config)
-            elif isinstance(config, Config):
-                self.config = copy.deepcopy(config)
-            elif isinstance(config, dict):
-                self.config = Config(config)
-        else:
-            self.config = Config({})
+    def _get_value_from_config(
```

Comment:
```suggestion
    def _get_value_from_updated_config(
```

I want to make sure that this function will return the final value extracted from the `positional_args + default_config`

Reply:
I'm not sure if this renaming makes sense, wouldn't it be more confusing? Of course the word config is confusing, but updated_config doesn't seem to be the correct word.